.changes/next-release/enhancement-dependency-2767.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "enhancement",
+  "category": "dependency",
```

Comment:
I think this has been `awscrt` in the past rather than a generic "dependency" category.

https://github.com/aws/aws-cli/blob/v2/.changes/2.13.35.json
https://github.com/aws/aws-cli/blob/v2/.changes/2.13.39.json#L123-L125
https://github.com/aws/aws-cli/blob/v2/.changes/2.14.2.json#L52-L55